dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -392,13 +396,10 @@ public synchronized void update(LogProviderConfig config) {
         commonConsoleLogHandlerUpdates();
 
         /*
-         * If messageFormat has been configured to 'basic' OR if consoleFormat is neither basic nor json (default to basic)
-         * - ensure that we are not connecting conduits/bufferManagers to the handler
+         * If messageFormat has been configured to 'basic' - ensure that we are not connecting conduits/bufferManagers to the handler
```

Comment:
was this comment change and change below intentional?  perhaps something related to the revert?